composer/callbacks/speed_monitor.py

```diff
@@ -90,13 +85,6 @@ def __init__(self, window_size: int = 100):
         self.loaded_state: Optional[Dict[str, Any]] = None
 
     def state_dict(self) -> Dict[str, Any]:
-        """Returns a dictionary representing the internal state of the SpeedMonitor object.
-
-        The returned dictionary is pickle-able via :func:`torch.save`.
-
-        Returns:
-            Dict[str, Any]: The state of the SpeedMonitor object
-        """
```

Comment:
why are these docstrings being deleted?

Reply:
Base class implementations of abstract methods generally do not need docstrings, as the abstract class should document the expected functionality.